.github/workflows/visual-regression-testing.yml

```diff
@@ -3,11 +3,13 @@ name: Visual Regression Tests
 on:
   pull_request_review:
     types: [submitted]
-
+  push:
+    branches:
+      - 'main'
 jobs:
   approved:
     name: Percy CI
-    if: github.event.review.state == 'approved'
+    if: github.event_name == 'push' || github.event.review.state == 'approved'
```

Comment:
I don't know yml format for GitHub workflow much.

Wondering if we need to update the above so they are two "jobs" triggering the same set of tasks.

Condition for Job 1) on push to `main` only
Condition for Job 2) when any PR gets a "approved" review

The first part of line 12 looks like push to any branch would trigger Percy?

Reply:
The condition will pass for any push but the workflow is only triggered for push to main.

Reply:
I can add an additional `&&` condition just make that clearer for readability.

Reply:
I would not want to duplicate the job, because then we need to keep that all in sync.

Reply:
> I can add an additional `&&` condition just make that clearer for readability. 

@tbrlpld  yes please. Thank you!

Reply:
@mmmavis I have updated the condition.